pkg/url/url.go

```diff
@@ -45,7 +46,7 @@ func (urls URLList) Get(urlName string) URL {
 // Delete deletes a URL
 func Delete(client *occlient.Client, kClient *kclient.Client, urlName string, applicationName string, urlType localConfigProvider.URLKind, isS2i bool) error {
 	if urlType == localConfigProvider.INGRESS {
-		return kClient.DeleteIngress(urlName)
+		return kClient.DeleteExtensionV1Beta1Ingress(urlName)
```

Comment:
Silly question. Why can't we keep `DeleteIngress` as-is and have this differentiation about various versions within `kclient`? Creating an exported function with this name and using it in `url` package feels like leaking things to me.

I could be wrong.

Reply:
I have modified the `DeleteIngress` method to be across both kinds of ingresses. 
I renamed to old function so as to not break the current flow

I will update usages to use new functions in my next PR, along with any tests that this might break just like i have done with listingress

Reply:
You mean `DeleteExtensionV1Beta1Ingress` will be renamed to `DeleteIngress` in next PR?

Reply:
> I will update usages to use new functions in my next PR, along with any tests that this might break just like i have done with listingress

Please leave a TODO for the time being